migrations/extras/extras.ts

```diff
@@ -18,7 +18,7 @@ export const linkableModelApiKeys = [
   'product_comparison',
   'changelog_entry',
   'showcase_project',
-  'success_story',
+  'case-study',
```

Comment:
These are Model API keys! We do not need to change them if we are only changing the URLs